static/src/javascripts/lib/easing.js

```diff
@@ -71,7 +71,4 @@ const create = (type: string, duration: number): Function => {
     };
 };
 
-export default {
-    functions,
-    create,
-};
+export { functions, create };
```

Comment:
Is it worth adding colour to these bland names?

```
export { easingFunctions, createEasing };
```

? I can see where this is going... a one-way ticket to Bloat Town. Stopping off at the Repetitive Coral and Redundancy Canyon 🌵